src/mango/test/user_docs.py

```diff
@@ -61,12 +61,11 @@ def setup_users(db, **kwargs):
 
 def setup(db, index_type="view", **kwargs):
     db.recreate()
-    db.save_docs(copy.deepcopy(DOCS))
     if index_type == "view":
         add_view_indexes(db, kwargs)
     elif index_type == "text":
         add_text_indexes(db, kwargs)
-
+    db.save_docs(copy.deepcopy(DOCS))
```

Comment:
What's the reason for changing this? Just that you didn't want to force the wait here?

Reply:
yes correct